### PR TITLE
docs: Clarify new ng workspace options

### PR DESCRIPTION
**Pull request by samhrncir**
Hope this helps make the tutorial go even smoother for people after me.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/start/start-deployment.md [hunk 1]
```diff
@@ -27,6 +27,29 @@ A best practice is to run your project locally before you deploy it. To run your
...
+    ```
+    Which stylesheet format would you like to use? (Use arrow keys)
+    ❯ CSS 
+    SCSS   [ https://sass-lang.com/documentation/syntax#scss                ] 
+    Sass   [ https://sass-lang.com/documentation/syntax#the-indented-syntax ] 
+    Less   [ http://lesscss.org                                             ] 
+    Stylus [ https://stylus-lang.com                                        ] 
+    ```
+    
+    Answer No, Yes, CSS respectivly to the above prompts.
```
Comment (aikidave) [inline aio/content/start/start-deployment.md:52]:
```suggestion
    This command displays a series of configuration prompts. For this tutorial, accept the default settings for each prompt.
```

I like where you are going with this! I think we should try to write it in a way that supports any future changes to `ng new`. Let me know if you have any questions about this suggestion!

**Review by aikithoughts — CHANGES_REQUESTED** [review]
Thanks for this suggestion! I provided an alternate wording that I think helps make the tutorial accurate and helpful, while also supporting any future changes to the `ng new` command. Let me know if you have any questions!

File: aio/content/start/start-deployment.md (hunk 1 above)
Comment (samhrncir) [reply aio/content/start/start-deployment.md]: @aikidave What is your understanding that the defaults are?  Opting Yes for stricter type checking causes the build to fail in `ng serve`.

To me, default conveys answering yes, and first options. What do you think?

Comment (samhrncir) [reply aio/content/start/start-deployment.md]: I prefer your wording, just hesitant on how _default_ will be interpreted :D

Comment (aikidave) [reply aio/content/start/start-deployment.md]: I usually think of defaults as whatever the capitalized option is. So for example:

```
Would you like to add Angular routing (y/N)
```

The default is No (`N`).

To put it another way: the defaults are whatever you get if you just hit the Enter key through all the prompts.

Hope this helps!

Comment (samhrncir) [reply aio/content/start/start-deployment.md]: This helps! Thanks, was stuck in my tunnel vision haha. Ill make the changes.

**Review by aikithoughts — APPROVED** [review]
LGTM!

Reviewed-for: global-docs-approvers

**Comment by aikidave** [comment]
Looking good! It looks like you need to modify the commit message. I'm guessing the capital C is throwing things off, but I'm not sure?

**Comment by aikidave** [comment]
Actually, I was able to rebase it for you. It's good to go!

Thank you again!

**Pull request closed without merging**